crates/sui-core/src/consensus_throughput_calculator.rs

```diff
@@ -469,6 +469,7 @@ mod tests {
     }
 
     #[test]
+    #[cfg_attr(msim, ignore)]
```

Comment:
Any test that manually creates an AuthorityMetrics fails `simtest` because of the code panics at this line where we initiate the start time: https://github.com/MystenLabs/sui/blob/ea5666b9c19141f34bb797d4b52c85b4e6a46441/crates/sui-core/src/metrics.rs#L105

So I turn all simtest off for those tests, and also it looks like they are all unit test.

Please let me know if there is a better way to fix this.

Reply:
Yes, unfortunately there is no way for simtest to rewrite or intercept regular `#[test]` tests, and if they do anything like checking the time, they will panic, because the simulator runtime has not been initialized.

You could just convert it to a `#[tokio::test]`, but adding the ignore is fine as well?

Reply:
I'll just leave it as it then :)